assets/css/lato.scss

```diff
@@ -0,0 +1,179 @@
+/*!
+  Lato font.
```

Comment:
I copied in the entire `lato.scss` file, made the necessary changes for this project, and commented out the fonts we aren't using. As much as I try to avoid committing commented code, this should be at low-risk of changing and sets the next developer up to only have to uncomment the code if we add another lato font to the project (like italics).